engine/table/src/main/java/io/deephaven/engine/table/impl/util/ServerStateStreamPublisher.java

```diff
@@ -0,0 +1,94 @@
+package io.deephaven.engine.table.impl.util;
+
+import io.deephaven.chunk.WritableChunk;
+import io.deephaven.chunk.attributes.Values;
+import io.deephaven.engine.table.ColumnDefinition;
+import io.deephaven.engine.table.TableDefinition;
+import io.deephaven.engine.table.impl.sources.ArrayBackedColumnSource;
+import io.deephaven.qst.type.Type;
+import io.deephaven.stream.StreamConsumer;
+import io.deephaven.stream.StreamPublisher;
+import io.deephaven.stream.StreamToBlinkTableAdapter;
+import io.deephaven.time.DateTimeUtils;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Objects;
+
+class ServerStateStreamPublisher implements StreamPublisher {
+
+    private static final TableDefinition DEFINITION = TableDefinition.of(
+            ColumnDefinition.ofTime("IntervalStartTime"),
+            ColumnDefinition.ofInt("IntervalDurationMicros"),
```

Comment:
Should we change to Duration on these types of columns? Can always present a view for compatibility.

Reply:
Unless we're doing it across the board, I don't think so. Also, we don't have great duration column sources.